scripts/classes/app-binary-configurator.js

```diff
@@ -96,12 +95,14 @@ class AppBinaryConfigurator {
   }
 
   configureLaunchScreen() {
-    const launchScreen = this.getLaunchScreenUrl();
-    const resizeConfig = this.binarySettings.launchScreen;
-    const production = this.config.production;
-    const launchScreenPath = './assets/launchScreen.png';
-    console.log('Configuring launch screen...');
-    return downloadAndResizeImage(launchScreen, launchScreenPath, resizeConfig, production);
+    return Promise.all(_.map(binarySettings, (settings, platform) => {
```

Comment:
I'd extract this `(settings, platform) => {` to separate function, perhaps that should be `confiogureLaunchScreen` and this method should be called `configureLaunchScreens`? Perhaps the same could be applied to `configureAppIcon` function too.  It will also help us have a better logging of what's happening, because we would have something like (I presume, correct me if I'm wrong):
> Configuring launch screens...
> Configuring launch screen for platform: ios
> Configuring launch screen for platform: android
> Launch screens configured.

etc.